perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -36,24 +44,60 @@
 	 * @param group Group JSON Group class
 	 * @return Group Newly created group
 	 */
+	/*#
+	 * Creates a new group in the specific VO defined by object vo in parameter.
+	 *
+	 * @param vo int Parent VO <code>id</code>
+	 * @param parentGroupId Integer parentGroupId
```

Comment:
Note existing javadoc ``Important: voId in object group is ignored.``. You should equally omit ``parentGroupId`` param for case when you are creating top-level group and passing ``vo`` param.